crates/red_knot_workspace/tests/check.rs

```diff
@@ -1,20 +1,20 @@
 use anyhow::{anyhow, Context};
 use red_knot_python_semantic::{HasTy, SemanticModel};
-use red_knot_workspace::db::RootDatabase;
-use red_knot_workspace::workspace::WorkspaceMetadata;
+use red_knot_workspace::db::ProjectDatabase;
+use red_knot_workspace::project::ProjectMetadata;
```

Comment:
The paths here are somewhat funny. `red_knot_project::project`. I need to think about how I can improve that but let's leave this to another PR